eloquentjs/2_3_chessboard.js

```diff
@@ -0,0 +1,20 @@
+function drawChessboard(width, height = width) {
```

Comment:
The `height = width` trick you're using is generally somewhat frowned upon. I'd recommend just having it be height and then adding a clause up top saying
```js
if (height === undefined) {
  height = width;
}
```